ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcConfigKeys.java

```diff
@@ -234,6 +248,28 @@ static void setLeaderOutstandingAppendsMax(RaftProperties properties, int maxApp
       setInt(properties::setInt, LEADER_OUTSTANDING_APPENDS_MAX_KEY, maxAppend);
     }
 
+    String LEADER_OUTSTANDING_INSTALL_SNAPSHOTS_MAX_KEY = PREFIX + ".leader.outstanding.install_snapshots.max";
+    int LEADER_OUTSTANDING_INSTALL_SNAPSHOTS_MAX_DEFAULT = 8;
+    static int leaderOutstandingInstallSnapshotsMax(RaftProperties properties) {
+      return getInt(properties::getInt, LEADER_OUTSTANDING_INSTALL_SNAPSHOTS_MAX_KEY,
+          LEADER_OUTSTANDING_INSTALL_SNAPSHOTS_MAX_DEFAULT, getDefaultLog(), requireMin(0));
+    }
+    static void setLeaderOutstandingInstallSnapshotsMax(RaftProperties properties, int maxInstallSnapshots) {
+      setInt(properties::setInt, LEADER_OUTSTANDING_INSTALL_SNAPSHOTS_MAX_KEY, maxInstallSnapshots);
+    }
+
+    String LEADER_INSTALL_SNAPSHOT_STREAM_TIMEOUT_KEY = PREFIX + ".leader.install_snapshot.stream.timeout";
```

Comment:
Let's use
- `.install_snapshot.request.element-limit`  //we may have a byte-limit later
- `.install_snapshot.request.timeout`

This should not be leader specific.  We can use it for Follower to Listener install-snapshot in the future.

Reply:
Thanks for the review! Made changes on the names.